### PR TITLE
Dispenser pick

### PR DESCRIPTION
**Pull request by SplinterGP**
## About The Pull Request
Adds a Dispenser for the engineer, currently it heals nearby robots and allows them to carry alot of items in, its equipped on backpack slot
i have bugtested this from past bugs if someone finds a new bug aweqewqd

## Why It's Good For The Game
Lets engineers drag around alot of items with them without being immediately accessible and also gives them a utility on healing ROBITS

## Changelog
:cl:
Coding By Splinter, Sprite by Zaha Sanko and title screen by nova_epee
add: Added the engineer dispenser which is equipped on back slot and carries alot of items, can only heal(robots) and access items when deployed
imageadd: new title screen announcing the new engineer dispenser
/:cl:

### CODE REVIEW

**Comment by SplinterGP** [comment]
checking dmidiff

**Comment by Lumipharon** [comment]
The storage capacity and max size means you can basically hold infinite stuff.

**Comment by ivanmixo** [comment]
Tasty powercreep

**Comment by Al-1ce** [comment]
Robots are essentially human+ without any of downsides of being human. No chems, no crit, no lasting damage, no broken bones, organs, etc. They ignore everything that has been implemented for balance.

Robots are meta as they are. I don't think the dispenser fits in right now. Not to mention the 30%+ damage malus to burn that they were intended to suffer from was axed completely.

**Comment by FosterGrime-Heymynameslouie** [comment]
finally, its been 4 years with 4 pr's trying to add dispenser all killed because they got stale

spdde mergh

**Comment by Losenis** [comment]
IT RETURNS

Please make it orderable from req, even if it's like 60 a pop. These things are extremely vulnerable to FF and crushers and the like so a way to get replacements would be most welcome ~~because gamers will sooner medevac, ghost, respawn as engi again, take dispenser, and loot their old body as well as new point stuff~~.

**Comment by Mannybrado** [comment]
big win for robot chads

**Comment by DerpiestBruh** [comment]
> Robots are essentially human+ without any of downsides of being human.

I mean you cannot process ANY chems at all, cannot chase xenos, cannot use modules nor req armor, is shredded by fire and healing takes long without external help.

I have bad feelings abt this PR removing the lack of regen for them, though

**Comment by FIXTHERADBOXPLEASE** [comment]
This is carefully balanced by the fact that I, every other Engineer, and probably half the PFCs will steal everything from the poor bastard's dispenser the moment it touches the ground. My experience in the test merges verifies this.

**Review by lbnesquik — COMMENTED** [review]
How do xenos interact with this? Can they slash it to disable it? How much health does it has? Can it be recovered if xenos disable it? Do bullets damage it?

File: sound/credits.md [hunk 1]
```diff
@@ -4,4 +4,8 @@ All sounds added/remixed from this PR https://github.com/tgstation/TerraGov-Mari
...
 They are used under the CC-BY-NC-SA 3.0 license.
 
-The sound used from this PR https://github.com/tgstation/TerraGov-Marine-Corps/pull/5604 is taken from https://freesound.org/people/nicStage/sounds/127731/ under CC Attribution
\ No newline at end of file
+The sound used from this PR https://github.com/tgstation/TerraGov-Marine-Corps/pull/5604 is taken from https://freesound.org/people/nicStage/sounds/127731/ under CC Attribution
+
+All the sound files on this [Pull Request](https://github.com/tgstation/TerraGov-Marine-Corps/pull/9494) are licensed under [CC-BY-NC-SA 3.0/Creative Commons Attribution-NonCommercial-ShareAlike 3.0 United States license](https://creativecommons.org/licenses/by-nc-sa/3.0/us/).
+The sound file dispenser_heal.ogg used on this same PR is taken from the game [Team Fortress 2](https://store.steampowered.com/app/440/Team_Fortress_2/) made by Valve. 
+The sound files dispenser_deploy.ogg and dispenser_undeploy.ogg used on this same PR was remixed from audio files from the game [Team Fortress 2](https://store.steampowered.com/app/440/Team_Fortress_2/) made by Valve.
```
Comment (TiviPlus) [inline sound/credits.md:11]: Wrong pr

**Review by TiviPlus — COMMENTED** [review]
Rest lgtm

File: sound/credits.md (hunk 1 above)
Comment (SplinterGP) [reply sound/credits.md:11]: oh ye lemme fix

**Comment by Losenis** [comment]
WHERE THE DISPENSER AT

**Pull request merged**